input/otlp/logs.go

```diff
@@ -239,3 +245,28 @@ func (c *Consumer) convertLogRecord(
 
 	return event
 }
+
+// https://www.elastic.co/guide/en/ecs/current/ecs-data_stream.html
+func sanitizeDataStream(k string) string {
```

Comment:
the functions should also be moved out of `logs.go`. It is not specific to logs.